packages/fx-core/src/common/azure-hosting/azureHosting.ts

```diff
@@ -88,11 +88,9 @@ export abstract class AzureHosting {
     module = AzureHosting.replacePluginId(module, pluginId);
 
     return {
-      Configuration: this.configurable
-        ? {
-            Modules: { [this.hostType]: module },
-          }
-        : undefined,
+      Configuration: {
```

Comment:
Any explanations for :
1. What is `this.configurable` for?
2. Why removing this flag?

Reply:
If the hosting is not configurable, it does not need to update bicep template.
https://github.com/OfficeDev/TeamsFx/blob/eec8ff4f80d64001d5438cb2a805ac35e0500cb8/packages/fx-core/src/common/azure-hosting/azureHosting.ts#L81